src/atris/management/commands/delete_old_historical_records.py

```diff
@@ -42,7 +50,15 @@ def handle(self, *args, **options):
                 msg=self.PARAM_ERROR,
             ))
             return
-        old_history_entries = HistoricalRecord.objects.older_than(days, weeks)
-        handled_entries_nr = old_history_entries.count()
-        old_history_entries.delete()
-        self.stdout.write('{} deleted.\n'.format(handled_entries_nr))
+        from_archive = options.get('from_archive')
+
+        try:
+            model_to_delete = ArchivedHistoricalRecord if \
```

Comment:
why not naming the variable `model_to_delete` to `model`? You are in the delete command, you know what it's usage is.
Why not use `.. = ... if from_archive else ...` - getting rid of `is True` - redundant.
Why keep this in try/except block? It's not necessary.

These 3 changes will make the line fit in 80 chars limit, thus getting rid of `\` annoyance.